steam/ext/dota2/protobufs/shared_enums.py

```diff
@@ -0,0 +1,79 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# sources: dota_shared_enums.proto
+# plugin: python-betterproto
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ....enums import IntEnum, classproperty
+
+if TYPE_CHECKING:
+    from collections.abc import Mapping
+
+    from typing_extensions import Self
+
+
+# fmt: off
+class DOTAGameMode(IntEnum):
+    NONE              = 0
+    AP                = 1
+    CM                = 2
+    RD                = 3
+    SD                = 4
+    AR                = 5
+    INTRO             = 6
+    HW                = 7
+    REVERSE_CM        = 8
+    XMAS              = 9
+    TUTORIAL          = 10
+    MO                = 11
+    LP                = 12
+    POOL1             = 13
+    FH                = 14
+    CUSTOM            = 15
+    CD                = 16
+    BD                = 17
+    ABILITY_DRAFT     = 18
+    EVENT             = 19
+    ARDM              = 20
+    MID1V1            = 21
+    ALL_DRAFT         = 22
+    TURBO             = 23
+    MUTATION          = 24
+    COACHES_CHALLENGE = 25
```

Comment:
These should be pascal case if they are more than 2 letters or not NONE (leave that as is)